common-content/en/module/how-our-curriculum-works/no-lectures/index.md

```diff
@@ -0,0 +1,11 @@
++++
+title="No lectures"
+headless="true"
+time= 30
+[objectives]
+    1="Explain why we don't lecture at CYF"
++++
+
+We don’t lecture trainees during live sessions. We are not against lectures; if you want to deliver a lecture, that’s great! Put it on Youtube and send us the link. Trainees can then watch lectures at their own pace, with captions, pause, and rewind enabled. Trainees also have free accounts kindly donated by Udemy and you can assign any Udemy for Business course.
```

Comment:
Late to the party, apologies, but perhaps we should provide some mechanism for sending us the lectures/lecture resources?